guides/micronaut-cloud-trace-base/create-app.adoc

```diff
@@ -0,0 +1,70 @@
+common:completesolution.adoc[]
+
+common:create-app-features.adoc[]
+
+=== OpenTelemetry
+
+The Micronaut framework uses https://opentelemetry.io/[OpenTelemetry] to generate and export tracing data.
+
+OpenTelemetry provides two annotations one to create a span and another to include additional information to the span.
+
+@WithSpan:: Used on methods to creates a new span using a given name or defaults to the method name.
+
+@SpanAttribute:: Used on method parameter to include the value to the span with a given name or defaults to the parameter name.
+
+The `@WithSpan` and `@SpanAttribute` can be used on non-private methods.
+
+If the annotations are not enough, or you want to add tracing to a private method, Micronaut’s tracing integration registers a `io.opentelemetry.api.trace.Tracer` bean which exposes the OpenTelemetry API and can be dependency-injected as needed.
+
+[NOTE]
+====
+The `io.micronaut.tracing.annotation` are available if you prefer or have an existing Micronaut Application already using them.
```

Comment:
I do not understand what this sentence means. Is this what you're trying to say?

```suggestion
The `io.micronaut.tracing.annotation` is also available if you prefer to use it or if you're working on an existing Micronaut application that's already already using it.
```
OR

```suggestion
The `io.micronaut.tracing.annotation`s are also available if you prefer to use them or if you are working on an existing Micronaut application that's already using them.
```